app_feup/lib/view/Widgets/NavigationDrawer.dart

```diff
@@ -32,14 +31,12 @@ class NavigationDrawerState extends State<NavigationDrawer> {
 
   getCurrentRoute() => ModalRoute.of(parentContext).settings.name == null ?
   drawerItems[0]:
-  ModalRoute.of(parentContext).settings.name.substring(1);
+  ModalRoute.of(parentContext).settings.name.substring(1) == 'ConfigurarParagens' ? 'Paragens' : ModalRoute.of(parentContext).settings.name.substring(1);
```

Comment:
If `ConfigurarParagens` is not a route you want to be able to access, it probably it should be used as an unnamed route.